### PR TITLE
Add tools to index.js

### PR DESCRIPTION
**Pull request by shehzan10**

### CODE REVIEW

File: tools/index.js [hunk 1]
```diff
@@ -1,6 +1,11 @@
 module.exports = {
+    extractB3dm : require('./lib/extractB3dm'),
+    extractCmpt : require('./lib/extractCmpt'),
+    extractI3dm : require('./lib/extractI3dm'),
     glbToB3dm : require('./lib/glbToB3dm'),
     glbToI3dm : require('./lib/glbToI3dm'),
     gzipTileset : require('./lib/gzipTileset'),
-    pipeline : require('./lib/runPipeline')
+    optimizeGlb : require('./lib/gzipTileset'),
```
Comment (lilleyse) [inline tools/index.js]: I don't see the need to expose this one as it is equivalent to just including `gltf-pipeline` as a library.

Comment (shehzan10) [inline tools/index.js]: Fixed. Also caught a copy-paste mistake in that. So thanks.

Comment (shehzan10) [inline tools/index.js]: Oops. Sorry changed the wrong thing.

File: tools/index.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
 module.exports = {
+    extractB3dm : require('./lib/extractB3dm'),
+    extractCmpt : require('./lib/extractCmpt'),
+    extractI3dm : require('./lib/extractI3dm'),
     glbToB3dm : require('./lib/glbToB3dm'),
     glbToI3dm : require('./lib/glbToI3dm'),
-    gzipTileset : require('./lib/gzipTileset'),
-    pipeline : require('./lib/runPipeline')
+    optimizeGlb : require('./lib/optimizeGlb'),
+    pipeline : require('./lib/runPipeline'),
```
Comment (lilleyse) [inline tools/index.js]: Better to match the naming. `pipeline` -> `runPipeline`.

**Comment by lilleyse** [comment]
`gzipTileset` shouldn't be removed from here.

**Pull request merged**